backend/tracim_backend/models/call.py

```diff
@@ -0,0 +1,68 @@
+import enum
+import typing
+
+from sqlalchemy import Column
+from sqlalchemy import Enum
+from sqlalchemy import ForeignKey
+from sqlalchemy import Integer
+from sqlalchemy import Sequence
+from sqlalchemy import String
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.orm import relationship
+
+from tracim_backend.models.auth import User
+from tracim_backend.models.meta import DeclarativeBase
+from tracim_backend.models.mixins import CreationDateMixin
+from tracim_backend.models.mixins import UpdateDateMixin
+
+
+class CallProvider(enum.Enum):
+    NONE = ""
+    JITSI_MEET = "jitsi_meet"
+
+
+class UserCallState(enum.Enum):
```

Comment:
Could be nice to document the meanings of the different states

Reply:
Should I mark this as resolved?

Reply:
Let's leave to @grignards the decision to mark it as resolved.

Reply:
I added comments to explain the states.